.gitignore

```diff
@@ -4,4 +4,5 @@ build
 bisq-pricenode
 bisq-pricenode.bat
 lib
+*.sh
```

Comment:
I like to make my own shell scripts in the project root and since its not used in the project and its annoying to get git asking to commit them all the time